_data/simple-icons.json

```
@@ -5934,6 +5934,11 @@
             "hex": "F7DF1E",
             "source": "https://cssinjs.org/"
         },
+        {
+            "title": "Juejin",
+            "hex": "007FFF",
+            "source": "https://xitu.juejin.cn"
```

```suggestion
            "source": "https://juejin.cn"
```

There's an SVG source in the header of the site now @LitoMore :)